integration-tests/utils.py

```diff
@@ -261,6 +261,14 @@ def all_units_ready(model):
                 yaml.dump(data, f)
         await model.deploy(bundle_dir)
     await wait_for_ready(model)
+    # In localhost we need proxy-extra-args="proxy-mode=userspace"
```

Comment:
Should we set this back to whatever the prev value was when we're done testing?

Reply:
No we just leave it like this. lxd deployments need to have this option set https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Deploying-on-LXD